### DISCUSSION TITLE
Minor broken link on https://asciinema.org/about

### DISCUSSION THREAD

**Discussion by mpassell** [asciinema/discussions#242] [General]
I just heard about asciinema from the Changelog Podcast. Sounds awesome! Tiny bug report: while looking around on asciinema.org, I noticed that the link to [Reza Ali's blog](http://www.syedrezaali.com/blog/) no longer works, so it would probably make sense to instead point it to his main page at https://www.syedrezaali.com/

**Comment by ku1ik** [comment]
Aha, good catch. Thanks!

**Reply by ku1ik** [reply]
Created issue in asciinema-server repo: https://github.com/asciinema/asciinema-server/issues/424

**Reply by mpassell** [reply]
You're welcome. I'm happy to help, even if it's in a small way. :)

**Comment by ku1ik** [comment]
I'm working on new documentation site for asciinema here https://github.com/asciinema/asciinema.github.io , I'll make sure link to Reza's blog is working.
